examples/quotes-api/cmd/quotes-api/wire.go

```diff
@@ -47,8 +48,9 @@ func initializeAPI() (*API, func(), error) {
 		log.LoggerProviderSet,
 		yeapi.ClientProviderSet,
 		quote.GetQuoteUseCaseProviderSet,
+		graph.ResolverProviderSet,
 		rest.QuoteControllerProviderSet,
-		server.ServerProviderSet,
+		server.GraphQLServerProviderSet,
 		wire.Value(server.Version("1.0.0")),
 		wire.Value([]httpds.Plugin(nil)),
 		ProvideHealthChecks,
```

Comment:
why does the example only use `GraphQLServerProviderSet`?
shouldn't we have 2 examples? one for GQL and one for rest?

Reply:
Sure, I'll clone this one and add it to `cmd/`